app/components/candidate_interface/continuous_applications/application_review_component.html.erb

```diff
@@ -1 +1,8 @@
 <%= render SummaryListComponent.new(rows: rows) %>
+
+<% if show_withdraw? %>
+  <div class="withdraw-your-application govuk-!-margin-bottom-6">
```

Comment:
Why the `withdraw-your-application` class?

Reply:
No reason, I'll remove it.